README.md

```diff
@@ -32,191 +32,9 @@ A user should be able to interact with the HTTP server as follows:
-        location: "/TO"
-      },
-      "/TO" => %{
-        handler: HandlerModule,
-        methods: ["METHOD"]
-      }
-  }
-```
-
-* `/FROM` is the original path on the server you are redirecting the client away from. This path will respond to the HTTP Client with a 301 status code, which will send the request to the path specified in the `location:` field.
-* `HTTPServer.Handlers.Redirect` is the module that lets you redirect the user to a different URL by sending an HTTP response with status 301.
-* `methods: ["METHOD"]` is a list of [HTTP request methods](https://en.wikipedia.org/wiki/HTTP#Request_methods) that the `/TO` path can respond to.
-* `location: "/TO"` is the path on the server you are redirecting the client towards. After receiving a response with a 301 status code at the `/FROM` endpoint, the `Location` response header will point toward this path as the new endpoint that will respond to the request.
-
-Please be sure that the `/TO` route and it's handler module is properly setup before redirecting the clients to that path.
-
-### An Example
-
-Let's say I want my app to respond to GET requests with "Hello World!" on the root route (/), also known as the application's home page.
-
-First, I'll create a new file called `hello_world.ex` in the `lib/handlers` folder. In that file I'll add my new handler module:
-
-```
-defmodule HTTPServer.Handlers.HelloWorld do
-  alias HTTPServer.Request
-  alias HTTPServer.Response
-  @behaviour HTTPServer.Handler
-
-  @impl HTTPServer.Handler
-  def handle(%Request{method: "GET"} = _req) do
-    body = "Hello World!"
-    {200, body, Response.build_headers(body)}
-  end
-end
-```
-
-Next I'll update the `lib/routes.ex` file to include the new route and it's corresponding handler:
-
-```
-defmodule HTTPServer.Routes do
-  alias HTTPServer.Handlers.HelloWorld
-  
-  def routes, 
-    do: %{
-      "/PATH" => %{
-        handler: HandlerModule,
-        methods: ["GET"]
-      } 
-    }
-end
-```
-
-What if I want my app to also respond to POST requests with the request body on the homepage? I would update my HelloWorld module to look like:
-
-```
-defmodule HTTPServer.Handlers.HelloWorld do
-  alias HTTPServer.Request
-  @behaviour HTTPServer.Handler
-
-  @impl HTTPServer.Handler
-  def handle(%Request{method: "GET"} = _req) do
-    body = "Hello World!"
-    {200, body, Response.build_headers(body)}
-  end
-  def handle(%Request{method: "POST"} = req) do
-    {200, req.body, Response.build_headers(req.body)}
-  end
-end
-```
-
-And finally update the `HTTPServer.Routes` module to reflect the newly added method:
-
-```
-defmodule HTTPServer.Routes do
-  alias HTTPServer.Handlers.HelloWorld
-  
-  def routes, 
-    do: %{
-      "/PATH" => %{
-        handler: HandlerModule,
-        methods: ["GET", "POST"]
-      } 
-    }
-end
-```
+For more information about connecting to the server as a client, basic routing, as well as handling redirects and static files, please see the doc folder.
```

Comment:
You should be able to link to other markdown files, 
`[I'm a relative reference to a repository file](../doc/another_file.md)`

Reply:
Ahhh see I pushed it up and added links to the url on the branch, but this is way better.